go.mod

```diff
@@ -0,0 +1,20 @@
+module github.com/go-debos/debos
```

Comment:
This can be merge seperately from introducing better tsting so pleas split it out